RealmSwift/Tests/TestCase.swift

```diff
@@ -113,6 +113,29 @@ class TestCase: XCTestCase {
         queue.sync { }
     }
 
+    /// Check whether two test objects are equal (refer to the same row in the same Realm), even if their models
+    /// don't define a primary key.
+    func assertEqual(_ o1: Object?, _ o2: Object?, fileName: String = #file, lineNumber: UInt = #line) {
+        if o1?.isSameObject(as: o2) ?? false {
```

Comment:
if `o1` is `nil` but `o2` is not, `assertEqual` won't trigger an XCTest failure.